server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec2.scala

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ons.addressIndex.server.modules
```

Comment:
These tests are unit tests for the QueryDefinition objects.
The other tests are for the response from ES.